test/onnx/test_fx_op_consistency.py

```diff
@@ -201,16 +201,6 @@
         "amin", dtypes=(torch.int16, *onnx_test_common.BOOL_TYPES),
         reason=onnx_test_common.reason_dynamo_does_not_support("ReduceMin", "bool, int16")
     ),
-    xfail(
-        "arange",
```

Comment:
this is failing in CI though, is there an update to `arange` between the onnxscript version bump?

Reply:
There is but some tests that should pass w/ the new version failed - so I am trying to figure that out